olio/service/resources/version_resource.go

```diff
@@ -31,10 +31,10 @@ func (vr *VersionResource) AddVersionExtractor(versionExtractor VersionExtractor
 func (vr *VersionResource) Init(r *gin.Engine) {
 	log.Debug("Setting up version resource.")
 
-	r.GET("/api/version", vr.getVersion)
+	r.GET("/api/version", vr.GetVersion)
```

Comment:
i wish there was a way to avoid artificially having to make private methods public just for the sake of testing them... have you done research on this end

Reply:
i could change it. using a different package is useful for unit tests to auto generate docs